torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -36,17 +36,16 @@ def horizontal_flip_bounding_box(
 ) -> torch.Tensor:
     shape = bounding_box.shape
 
-    # TODO: Investigate if it makes sense from a performance perspective to have an implementation for every
-    #  BoundingBoxFormat instead of converting back and forth
-    bounding_box = convert_format_bounding_box(
-        bounding_box.clone(), old_format=format, new_format=features.BoundingBoxFormat.XYXY, inplace=True
-    ).reshape(-1, 4)
+    bounding_box = bounding_box.clone().reshape(-1, 4)
```

Comment:
Here, there is an implicit "bug fix" or feature when previously we were transforming CXCYWH into XYXY and back with long dtype we had +/- 1 data loss. This does not happen here anymore...